apps/web/src/styles/global.css

```
@@ -58,14 +62,28 @@ html {
   font-family: var(--font-sans);
   font-size: 16px;
   line-height: 1.5;
+  letter-spacing: -0.005em;
   text-rendering: optimizeLegibility;
```

Stylelint (17.7.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)